src/objective-c/tests/run_plugin_tests.sh

```diff
@@ -24,9 +24,8 @@ cd $(dirname $0)
 
 ROOT_DIR=../../..
 BAZEL=$ROOT_DIR/tools/bazel
-BAZEL_EXEC_ROOT=$ROOT_DIR/bazel-out/darwin-fastbuild/bin
-PROTOC=$BAZEL_EXEC_ROOT/external/com_google_protobuf/protoc
-PLUGIN=$BAZEL_EXEC_ROOT/src/compiler/grpc_objective_c_plugin
+PROTOC=bazel-bin/external/com_google_protobuf/protoc
```

Comment:
You probably still need `$ROOT_DIR` here

Reply:
Good catch! Fixed.